homeassistant/components/websocket_api/commands.py

```diff
@@ -202,15 +203,16 @@ async def handle_call_service(
 
     try:
         context = connection.context(msg)
-        await hass.services.async_call(
+        response = await hass.services.async_call(
             msg["domain"],
             msg["service"],
             msg.get("service_data"),
             blocking,
             context,
             target=target,
+            return_response=msg["return_response"],
         )
-        connection.send_result(msg["id"], {"context": context})
+        connection.send_result(msg["id"], {"context": context, "response": response})
```

Comment:
Should we name the response item just `"response"` as it is now or rename to `"service_response"` or even `"service_response_data"`?